Jenkinsfile

```diff
@@ -199,7 +199,7 @@ node( 'rocmtest' )
     // If this a clang_tot_upgrade build, kick off downstream hip build so that the two projects are in sync
     if( env.BRANCH_NAME.toLowerCase( ).startsWith( 'clang_tot_upgrade' ) )
     {
-      build( job: 'ROCm-Developer-Tools/HIP/master', wait: false )
+      build( job: 'ROCm-Developer-Tools/HIP/master', wait: true )
```

Comment:
The only comment here is that without the boolean parameter, the hip job will launch all 3 build configurations (hcc-tot,hcc-rocm,nvcc).  I think this is what we want when clang_tot_upgrade branch changes.